src/Components/App.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import Navbar from './Navbar';
+import Countries from './Countries';
+import CountryInfo from './CountryInfo';
+import { dataLoading, loadDataThunk, selectData } from '../redux/covid/covid';
+import countriesNames from './countriesNames';
+import '../styles/app.css';
+import Footer from './Footer';
+
+function App() {
```

Comment:
Look at the images below. Here, we are trying to build a **mobile web app**. **NOTE:** This problem occurs in your **country** route as well.
- [ ] On the first image, your App stops being responsive from about **555px** of screen width.
- [ ] On the second image, your App also stops being responsive from about **802px**. Kindly fix your responsiveness. You could use CSS grid or flexbox for your styling.

![redwing-resp](https://user-images.githubusercontent.com/36542986/142624790-b82f6d99-9306-482d-bcfc-c879f6cd7983.jpg)

![redwing-resp-2](https://user-images.githubusercontent.com/36542986/142624797-ba49799f-7abe-483b-b7cf-ff5ca657a2f5.jpg)